src/systems/environment_preload/EnvironmentPreload.cc

```diff
@@ -93,32 +93,52 @@ void EnvironmentPreload::PreUpdate(
       return;
     }
 
+    components::EnvironmentalData::ReferenceUnits unit{
+      components::EnvironmentalData::ReferenceUnits::RADIANS};
     std::string timeColumnName{"t"};
+    bool ignoreTime = false;
     std::array<std::string, 3> spatialColumnNames{"x", "y", "z"};
     auto spatialReference = math::SphericalCoordinates::GLOBAL;
-
     sdf::ElementConstPtr elem =
         this->dataPtr->sdf->FindElement("dimensions");
     if (elem)
     {
+      if (elem->HasElement("ignore_time"))
+      {
+        ignoreTime = elem->Get<bool>("ignore_time");
+      }
```

Comment:
@arjo129 meta: OK with the change in the interest of time, but this is odd. Why force the user to provide a data file with a timestamp column (that we even have to specify on load) if it won't be used?